setup.py

```diff
@@ -0,0 +1,51 @@
+import os
+import subprocess
+import multiprocessing
+from distutils.core import setup
+from distutils.errors import LibError
+from distutils.command.build import build as _build
+
+class build(_build):
+    @staticmethod
+    def _configure():
+        if subprocess.call(['./configure']) != 0:
```

Comment:
Also it would probably be better to use `cwd=dirname(__file__)` rather than assuming the setup.py is run from its containing directory.